tools/packaging/kata-deploy/helm-chart/kata-deploy/values.yaml

```diff
@@ -0,0 +1,19 @@
+imagePullPolicy: Always
+imagePullSecrets: []
+image:
+  reference: quay.io/kata-containers/kata-deploy
+  tag: ""
```

Comment:
If tag is empty we will per default use chart.version which translates  to VERSION